tests/functions_tests/test_loss.py

```diff
@@ -0,0 +1,38 @@
+# Currently, the test for SoftmaxCrossEntropy is disabled since onnxruntime
```

Comment:
Could you create a bug and add its URL here?

Reply:
Sorry, what do you mean "create a bug and add its URL here"?

Reply:
Sorry, I meant an issue: https://github.com/chainer/onnx-chainer/issues/new

Reply:
Understood.